server/routes/ssot.js

```diff
@@ -7,10 +7,12 @@ const parsingController = require('../controllers/ssotParsingController');
 const retrievalController = require('../controllers/ssotRetrievalController');
 
 /// SSOTPARSING ROUTES ///
-
 router.get('/parser/get-robot-code', parsingController.getRobotCode);
 router.get('/get/:id', retrievalController.getSingleSourceOfTruth);
 router.get('/getAvailableBotsForUser/:userid', retrievalController.getBotList);
 router.get('/renameBot', retrievalController.renameBot);
+router.get('/retrieveMetadataForBot/:botId', retrievalController.retrieveBotMetadata);
+router.get('/shareBotWithUser', retrievalController.shareBotWithUser);
+router.get('/createNewBot', retrievalController.createNewBot);
```

Comment:
Can we maybe think of new names here? Something like /bot/new and /bot/edit/rename would be good for categorizing the routes

Reply:
As mentioned in one of my comments below I think we should also use 'robot' or 'bot' consistently

Reply:
This is part of issues #155 , to build such a schema that allows uniform and easier accesses to our API / backend.